code/modules/mining/machine_vending.dm

```diff
@@ -5,6 +5,7 @@
 	desc = "An equipment vendor for miners, points collected at an ore redemption machine can be spent here."
 	icon = 'icons/obj/machines/mining_machines.dmi'
 	icon_state = "mining"
+	var/icon_deny = "mining-deny"
```

Comment:
put this var below `circuit` so there's no new var declarations between already declared vars